src/gam/gapi/cloudidentity/groups.py

```diff
@@ -222,16 +229,30 @@ def print_():
                 display.add_titles_to_csv_file([
                     'Owners',
                 ], titles)
-    gam.printGettingAllItems('Groups', None)
+    gam.printGettingAllItems('Groups', usemember)
     page_message = gapi.got_total_items_first_last_msg('Groups')
-    entityList = gapi.get_all_pages(ci.groups(),
-                                    'list',
-                                    'groups',
+    if usemember:
```

Comment:
the challenge here is searchTransitiveGroups returns GroupRelation:

https://cloud.google.com/identity/docs/reference/rest/v1beta1/groups.memberships/searchTransitiveGroups#response-body

while groups.list just returns Groups:
https://cloud.google.com/identity/docs/reference/rest/v1beta1/groups/list#response-body

these two have some differences today but if that difference grows we end up with a whole bunch of if/then/else logic on this and it's just messy. searchTransitiveMemberships is similar.

I think it'd be cleaner to break these transitive API calls out into their own funciton (and thus own GAM command). The functions will be cleaner and easier to read and it means admins need to call another GAM command to use the transitive functions. Considering that the transitive functions only work on the Enterprise SKUs this makes sense also that some GAM commands won't work (as opposed to some arguments on some GAM commands not working).